common/route/awooter/awooter.h

```diff
@@ -16,15 +16,26 @@
  *  OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  *
  */
-
+#pragma once
 #ifndef AWOOTER_H
 #define AWOOTER_H
 
+#include "cxx.h" // this is the unsolved piece of the riddle
 #include "nextpnr.h"
 NEXTPNR_NAMESPACE_BEGIN
 
+typedef dict<IdString, std::unique_ptr<NetInfo>> NetDict;
+
 extern bool router_awooter(Context* ctx);
 
+uint64_t npnr_netinfo_users_leak(NetInfo &net, rust::Vec<PortRef> &vec) {
+    for (auto& item : net.users) {
+        vec.push_back(item);
+    }
+    vec.shrink_to_fit();
+    return vec.size();
+}
+
```

Comment:
This kind of makes me uncomfortable; could you not just provide a prototype and keep the definition inside `awooter.cc`?

Reply:
It apparently did not need to be exposed at all? Unsure. May revisit this after the rebase.